Assets/Scripts/Turretfunctions/SingleShot.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections;

```

Comment:
It's unclear what this script is for

Reply:
oh whoops I thought I deleted that script.